apps/src/templates/projects/PersonalProjectsTableActionsCell.jsx

```diff
@@ -45,42 +54,73 @@ class PersonalProjectsTableActionsCell extends Component {
     this.props.showDeleteDialog(this.props.projectId);
   };
 
+  onRename = () => {
+    this.props.startRenamingProject(this.props.projectId, this.props.updatedName);
```

Comment:
it looks like `startRenamingProject` gets used in two different ways. In this instance you are using it when you start editing the project name, which is great since it matches the name of the function...